source/_posts/2019/2019-11-18-how-to-delegate-code-reviews-to-ci.md

```diff
@@ -71,20 +71,47 @@ vendor/bin/rector process src --config rector-ci.yaml --dry-run
 
 I've prepared a demo with PHP code and a testing pipeline for all widely used CI services.
 
-- [Github + Travis CI](#1-github-travis-ci)
+- [GitHub Actions](#0-github-actions)
+- [GitHub + Travis CI](#1-github-travis-ci)
 - [Gitlab CI](#2-gitlab-ci)
 - [BitBucket](#3-bitbucket)
 
 <br>
 
-- ~~Github Actions~~
-*Do you know how to work with Github Actions? Please let me know how would script look like, so we can complete the list.*
+There you'll find all the configuration you need to **let your CI do code-reviews**.
+
+### 0. GitHub Actions
 
 <br>
 
-There you'll find all the configuration you need to **let your CI do code-reviews**.
+<a href="http://github.com/tomasvotruba/rector-ci-demo" class="btn btn-info">Repository</a>
+<a href="https://github.com/TomasVotruba/rector-ci-demo/actions/" class="btn btn-success ml-3">CI Feedback</a>
```

Comment:
The link needs to lead exatly to failing CI line with the Rector diff.

E.g. https://travis-ci.com/TomasVotruba/rector-ci-demo/jobs/258286278#L320

Reply:
Ok, the new commit (37207bf) links to the line; as the [PR](https://github.com/TomasVotruba/rector-ci-demo/pull/1) is not merged yet, I could only link to my fork to demonstrate how that can work.

Reply:
Thanks :+1: The link is exactly what I meant